src/version.c

```diff
@@ -96,13 +96,12 @@ int get_current_version(char *path_prefix)
 	FILE *file;
 	int v = -1;
 	char *buildstamp;
-	char *src, *dest;
 
-	string_or_die(&buildstamp, "%s/usr/lib/os-release", path_prefix);
```

Comment:
The changes to this function seem unrelated... You already added a new function to read a mix version file. That seems appropriate, but I think Clear Linux's version should still be read from /usr/lib/os-release, since that is the distro-agnostic location to include such metadata.